lib/ios/ios.nit

```diff
@@ -17,3 +17,21 @@ module ios
 
 import platform
 import app
+
+import cocoa::foundation
+
+redef fun print(msg) do msg.to_s.nslog
+redef fun print_error(msg) do msg.to_s.nslog
+
+redef class Text
+	private fun nslog do to_nsstring.nslog
+end
+
+redef class NSString
+	private fun nslog in "ObjC" `{ NSLog(@"%@", self); `}
+end
+
+redef class NativeString
+	# FIXME temp workaround for #1945, bypass Unicode checks
+	redef fun char_at(pos) do return self[pos].ascii
```

Comment:
Since this is a workaround, this might work temporarily, but this looks seriously broken on non-ASCII characters